src/components/ProjectSearch/TextSearch.js

```diff
@@ -153,28 +160,23 @@ export default class TextSearch extends Component {
       }
     });
 
-    return (
-      <span className="line-value">
-        {matches}
-      </span>
-    );
+    return dom.span({ className: "line-value" }, ...matches);
```

Comment:
Good question.  There is an issue where `<span>{matches}</span>` causes a react
error that keys should be unique,.. oh! I see jasonLaster just commented about that :)